src/site/content/en/fast/fast.11tydata.js

```diff
@@ -79,6 +79,7 @@ module.exports = {
           'performance-budgets-101',
           'your-first-performance-budget',
           'incorporate-performance-budgets-into-your-build-tools',
+          'use-lighthouse-for-performance-budgets',
```

Comment:
For some reason, I'm not seeing this new article in the fast path. I've pinged Rob to check if I'm just doing something wrong in the staging. The styles aren't showing up either, so it must be something I'm doing wrong.

Reply:
Ah, you are missing yaml information in the article.